tests/PySys/environments/environment_c8y.py

```diff
@@ -10,7 +11,63 @@
 """
 
 
+class Cumulocity(object):
+
+    c8y_url = ""
+    tenant_id = ""
+    username = ""
+    password = ""
+    auth = ""
+
+    def __init__(self, c8y_url, tenant_id, username, password):
+        self.c8y_url = c8y_url
```

Comment:
Since these values now can be empty (change in the xml), I would expect that we should check if they really contain a valid value. That would help to find out if necessary parameters are not set during the test

Reply:
Yes. I'll add that check so that the devs get the right feedback.